### ISSUE TITLE
exit 1 without evidence of an error and empty output

### ISSUE THREAD

**Issue by berga** [lucagrulla/cw#265]
**Describe the bug**
The command `cw ls groups` exits with code 1 without output

It seems a duplicate of #217 but after applying the workaround I cannot find out 
which configuration problem would cause the issue.

**Expected behavior**
In case of a configuration error it would be nice to have an error output at least.

**Desktop (please complete the following information):**
 - OS: Ubuntu on WSL
 - Terminal zsh
 - cw version 4.1.3

**Additional context**
I've tried the debugging flag without any success.
```
cw [debug] 2024/06/26 12:22:39 Debug mode is on. Will print debug messages to stderr
cw [debug] 2024/06/26 12:22:39 awsProfile: , awsRegion: eu-west-1 endpoint:
```

**Comment by gene1wood** [comment]
Agreed. When the tool exits non-zero, but gives no error message, it leaves the user stuck and unable to troubleshoot what the problem is.
